nimble/transport/pkg.yml

```diff
@@ -52,9 +52,7 @@ pkg.deps.BLE_MONITOR_RTT:
 
 pkg.init:
     ble_transport_init: 250
-    ble_transport_hs_init:
-        - $after:ble_transport_init
-        - $before:ble_transport_ll_init
+    ble_transport_hs_init: 251
```

Comment:
we want as much fixed stages as possible so proper ordering of ble_svc_gap should be forced by adding proper dependencies there

Reply:
That would be nice, but currently all the services use hardcoded (via syscfg.h) stages. I could set GAP to be after transport_hs, but then GATT will need to be changed, etc etc.

Reply:
yes, lets hotfix this for now and do proper dependencies on all services after release